package.json

```diff
@@ -73,7 +73,7 @@
     "jest": "24.9.0",
     "lerna": "^3.20.2",
     "lint-staged": "^10.0.8",
-    "next": "^9.3.0",
+    "next": "^9.3.7",
```

Comment:
Thanks for the PR! Any reason not to just set this to the canary version? `9.3.7-canary.10`

Reply:
Can do that - just thought it would be weird to commit a canary version. Would it be better to just do that and get on with it?

Reply:
I'm fine to do this. Also, we need to pin the next.js version. It shouldn't be a range.